util/cache/cache.go

```diff
@@ -23,27 +23,14 @@ func NewCache(client CacheClient) *Cache {
 // AddCacheFlagsToCmd adds flags which control caching to the specified command
 func AddCacheFlagsToCmd(cmd *cobra.Command) func() (*Cache, error) {
 	redisAddress := ""
-	sentinelAddresses := make([]string, 0)
```

Comment:
Please keep the ability to use failover clients. It should provide a quick way switching back to sentinel if HA proxy is not stable.